### ISSUE TITLE
Failures on Add an initial tests for dartdev that can execute and pass on the Dart build bots, as well as locally.

### ISSUE THREAD

**Issue by jwren** [dart-lang/sdk#40318]
@whesse -- how do I disable all dart2js tests for dartdev?

There are new test failures on [Add an initial tests for dartdev that can execute and pass on the Dart build bots, as well as locally.](https://github.com/dart-lang/sdk/commit/e9bd2595f4d913bbaf7bd44365fc8bee0075cf15).

The tests
```
pkg/dartdev/test/command_test RuntimeError (expected Pass)
```
are failing on configurations
```
dart2js-minified-linux-d8
```

**Comment by sigmundch** [comment]
I believe skips are still handled by the old status files.

In this case, you can edit pkg/pkg.status and add:
```
pkg/dartdev/*: SkipByDesign
```

under the `[ $compiler == dart2js ]` section.

**Comment by sigmundch** [comment]
closed as a dupe of #40319
